.github/workflows/build.yml

```diff
@@ -33,6 +34,7 @@ jobs:
         run: |
           pacman --sync --noconfirm --needed make mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake
           pacman --sync --noconfirm --needed mingw-w64-x86_64-fltk mingw-w64-x86_64-libjpeg-turbo mingw-w64-x86_64-gnutls mingw-w64-x86_64-pixman
+          pacman --sync --noconfirm --needed mingw-w64-x86_64-ffmpeg
```

Comment:
`H264_LIBS` defaults to `WIN` when building for Windows platform, which makes MediaFoundation H.264 decoder is preferred over libav. Does this mean `mingw-w64-x86_64-ffmpeg` is unused for `build-windows` job?